test/dynamo/test_misc.py

```diff
@@ -3043,6 +3043,81 @@ def f(pred, x):
         b = opt_fn(torch.tensor(True), torch.tensor([0.25, 0.25]))
         self.assertTrue(same(torch.sin(torch.tensor([0.25, 0.25])), b))
 
+    def test_nonzero_static(self):
```

Comment:
Instead of doing these tests by hand, you should write an opinfo for nonzero_static (it will be similar to the opinfo for nonzero, see torch/testing/_internal/common_methods_invocations.py grep for nonzero), and then you will get a number of tests for free. It's a little non-standard how to specify non-aten ops, but if you move nonzero_static to prims namespace, then you can follow the prims examples here.

Reply:
Cool! Will replace the manual test cases with `opinfo`.

Reply:
Get back to this today. Will write an `opinfo` for `nonzero_static`. Two questions:
 1. How to run the specific test (where is the test file?) for my op or `nonzero()`? I want to test it locally.
 2. There are bunch of [skipped cases](https://github.com/pytorch/pytorch/blob/477f3f555f2a7953a1d2c56b732ce59aca46ae15/torch/testing/_internal/common_methods_invocations.py#L16754-L16767) for `nonzero()`, how do I know what to skip for `nonzero_static()`?

Reply:
FYI, will remove this test later once able to verify all test cases here are covered by `opinfo`

Reply:
Chatted with Ed. OpInfo is not a full replacement of manual unit tests. We still need manual tests to ensure the correctness of the op. So will keep the tests and and OpInfo